docs/cli/Ignite-CLI.md

```diff
@@ -98,6 +98,7 @@ Starts the interactive prompt for generating a new Ignite project. Any options n
 - `--overwrite` overwrite the target directory if it exists
 - `--targetPath` string, specify a target directory where the project should be created
 - `--removeDemo` will remove the boilerplate demo code after project creation
+- `--mst` flag to specify whether to include MobX-State-Tree in project
```

Comment:
I think we should document that this only works when also using `--removeDemo`, I was confused by this at first